src/test/cli.up.test.ts

```diff
@@ -45,6 +45,61 @@ describe('Dev Containers CLI', function () {
 			await shellExec(`docker rm -f ${containerId}`);
 		});
 
+		it('should execute successfully with valid config and dotfiles with custom install path as filename', async () => {
```

Comment:
I think we forgot to move the existing tests over: https://github.com/microsoft/vscode-remote-containers/blob/0be791aae106ec5bb5e1c57fd9edf13b62f4d18a/test/core/dotfiles.test.ts#L23

Reply:
Thanks!

Reply:
I refactored and wrote several new tests under dotfiles.test.ts.  I think this is solid coverage and moving over this test verbatim doesn't add any additional value.